script/setup/install-cni

```diff
@@ -28,13 +28,13 @@ CNI_CONFIG_DIR=${DESTDIR}/etc/cni/net.d
 go get -d github.com/containernetworking/plugins/...
 cd "$GOPATH"/src/github.com/containernetworking/plugins
 git checkout $CNI_COMMIT
-FASTBUILD=true ./build.sh
+./build_linux.sh
 sudo mkdir -p $CNI_DIR
 sudo cp -r ./bin $CNI_DIR
 sudo mkdir -p $CNI_CONFIG_DIR
 cat << EOF | sudo tee $CNI_CONFIG_DIR/10-containerd-net.conflist
 {
-  "cniVersion": "0.3.1",
```

Comment:
can we push that to containerd/cri as well and test it there...  is probably ok but until we get cri builds running again and / or merged into container/containerd I think cri buckets (e2e) may need to be run.

Reply:
This PR is a cherry pick from master (it was labeled for cherry picking); I'm not too familiar with these; could you open a PR on master (if needed), then I can cherry-pick it here as well? 😅🤗

Reply:
started... https://github.com/containerd/cri/pull/1573  I'll cherry that as well to the service for containerd/cri .. and open a pull an cherry for the cri update over here on containerd...